backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/database/Setup.kt

```diff
@@ -0,0 +1,10 @@
+package app.ehrenamtskarte.backend.migration.database
+
+import org.jetbrains.exposed.sql.SchemaUtils
+import app.ehrenamtskarte.backend.migration.database.Migrations
+
+fun setupDatabase() {
+    SchemaUtils.create(
+        Migrations,
+    )
+}
```

Comment:
This step does not make sense anymore, since the schema already has been created in the `createTableIfNotExists()` function during migrations, right?